### PR TITLE
STR-23 -- BE: Setup and routing for generic email sending

### PR DESCRIPTION
**Pull request by liadGolan**
# Description
Sets up email sending so that emails can be sent
(please note that no tests where added to this since this just ensuring that adding this package works)
  
# Testing
(please note that no unit tests where added to this since this just ensuring that adding this package works)

1.) visit `/send_mail_test`
2.) you should see and html page that says good
3.) check your email for an email

# Installation
`composer install`
ensure .env is up to date

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -20,4 +20,5 @@
 
 Route::get('/get_group/{user_id}','UserGroupController@getGroup');
 
-Route::get('/user','UserRoleController@sortAuthenticatedUsers');
\ No newline at end of file
+Route::get('/send_mail_test','MailController@sendMail');
+Route::get('/user','UserRoleController@sortAuthenticatedUsers');
```
Comment (frlzjosh) [inline routes/web.php]: this should be `UserGroupController`

**Review by frlzjosh — DISMISSED** [review]
real small change

**Review by frlzjosh — APPROVED** [review]
👍

File: routes/web.php [hunk 2]
```diff
@@ -20,4 +20,5 @@
 
 Route::get('/get_group/{user_id}','UserGroupController@getGroup');
 
-Route::get('/user','UserRoleController@sortAuthenticatedUsers');
\ No newline at end of file
+Route::get('/send_mail_test','MailController@sendMail');
```
Comment (luisjg) [inline routes/web.php]: URLs should be using kebab case

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**